doc/user_guide/dynamo0p3.rst

```diff
@@ -1099,10 +1104,10 @@ and the following stencil (with ``extent=2``):
 .. code-block:: none
 
   |   |   | 9 |   |   |
-  |   |   | 5 |   |   |
-  | 6 | 2 | 1 | 3 | 7 |
-  |   |   | 4 |   |   |
   |   |   | 8 |   |   |
+  | 3 | 2 | 1 | 6 | 7 |
+  |   |   | 4 |   |   |
+  |   |   | 5 |   |   |
```

Comment:
If I've understood the changes correctly, it seems that the way one of the existing stencil types is specified has changed too? Please could you update the rules on Subroutine arguments section (p.66 of the User Guide).

Reply:
In particular, I see that the stencil-size argument can now be an array lookup.

Reply:
I've updated the info on page 66 to give more detail on the kernel subroutine arguments and their purpose.